xml_converter/src/packaging_protobin.cpp

```diff
@@ -214,14 +214,8 @@ void _write_protobuf_file(
     outfile.close();
 }
 
-void write_protobuf_file(
-    const string& marker_pack_root_directory,
-    const StringHierarchy& category_filter,
-    const map<string, Category>* marker_categories,
-    const vector<Parseable*>* parsed_pois) {
+std::map<string, std::vector<Parseable*>> _construct_category_to_pois_map(const vector<Parseable*>* parsed_pois) {
```

Comment:
I dont think this function needs an underscore prefix just for being a non-public function.  It would in python but this is C++.

Reply:
I see. In C++ its more used for helper functions for functions of that name, correct?

Reply:
Correct, a good example of how I use it is for recursive functions that need to accept more than I want to expose to a user in order to recurse.